src/player_hardcoded_effects.cpp

```diff
@@ -829,13 +829,15 @@ static void eff_fun_hypovolemia( Character &u, effect &it )
                     warning = _( "You are anxious and cannot collect your thoughts." );
                     u.mod_focus( -rng( 1, u.get_focus() * intense / it.get_max_intensity() ) );
                     break;
-                case 4:
+                case 4: {
```

Comment:
I guess adding this bracket isn't necessary.

Reply:
I'm declaring a new variable within the case statement, so the scope is required.